<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rider_make_id</class>
 <widget class="QWidget" name="rider_make_id">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="back_to_login">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>뒤로</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LOGO_1">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>31</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="rider_img.qrc">:/rider_img/coucou_logo_width.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>362</width>
            <height>605</height>
           </rect>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>9</y>
             <width>341</width>
             <height>581</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="bottomMargin">
             <number>50</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>회원 정보를 입력해주세요</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_7">
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <widget class="QLineEdit" name="insert_pnum">
               <property name="geometry">
                <rect>
                 <x>60</x>
                 <y>10</y>
                 <width>271</width>
                 <height>41</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_6">
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <widget class="QLineEdit" name="insert_pw">
               <property name="geometry">
                <rect>
                 <x>60</x>
                 <y>10</y>
                 <width>271</width>
                 <height>41</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <widget class="QLineEdit" name="insert_name">
               <property name="geometry">
                <rect>
                 <x>60</x>
                 <y>10</y>
                 <width>271</width>
                 <height>41</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <widget class="QLineEdit" name="insert_PI">
               <property name="geometry">
                <rect>
                 <x>60</x>
                 <y>10</y>
                 <width>271</width>
                 <height>41</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <widget class="QLineEdit" name="insert_addr">
               <property name="geometry">
                <rect>
                 <x>60</x>
                 <y>10</y>
                 <width>271</width>
                 <height>41</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <widget class="QLineEdit" name="insert_motor_num">
               <property name="geometry">
                <rect>
                 <x>70</x>
                 <y>10</y>
                 <width>261</width>
                 <height>41</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <widget class="QLineEdit" name="insert_local">
               <property name="geometry">
                <rect>
                 <x>70</x>
                 <y>10</y>
                 <width>261</width>
                 <height>41</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="okay_btn">
              <property name="text">
               <string>회원가입 </string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rider_img.qrc"/>
 </resources>
 <connections/>
</ui>
